DSESwift/DataStorageEngine.swift

```diff
@@ -31,12 +30,12 @@ public class DataStorageEngine{
     // reference to the datasyncer
     var dataSyncer = DataSyncer()
```

Comment:
@kanekin i'm not sure, but i thought by set it as internal, it will be accesible from the whole module, but by set it as private, it only accessible from the DataStorageEngine class.